src/cache/cache_las.c

```diff
@@ -1237,9 +1238,19 @@ __wt_las_sweep(WT_SESSION_IMPL *session)
         /*
          * Remove the previous obsolete entries whenever the scan moves into next key or the current
          * LAS record is not a modify record.
+         *
+         * The special ondisk value that is present in the LAS is only removed whenever a next
+         * record of the same key gets removed. This is to protect the cases where we may still need
+         * the older ondisk value for some scenarios. The draw back is that this record never be
+         * removes until no updates happens on that key, but these scenarios are minimal.
```

Comment:
removes until no updates --> removed until more updates

Reply:
Can you also comment on what the special ondisk value is, ie one with 0 txnid and 0 timestamp that is globally visible and always the oldest record. It represents a previous on-disk value that is still needed for an old reader

Reply:
Changed accordingly.